src/base/projects/ProjectContentRoutes.svelte

```diff
@@ -15,26 +15,27 @@
   export let content: ProjectContent;
   export let revision: string;
   export let path: string;
+  export let peer: string;
 </script>
 
 <Router>
   <!-- The default action is to render Browser with the default branch head -->
   <Route path="/">
-    <Browser {urn} {org} {user} {config} {tree} {project} {branches}
+    <Browser {urn} {org} {user} {config} {tree} {project} {branches} {peer}
       locator={project.head}
       bind:content={content}
       bind:path={path}
       bind:revision={revision} />
   </Route>
   <Route path="/tree">
-    <Browser {urn} {org} {user} {config} {tree} {project} {branches}
+    <Browser {urn} {org} {user} {config} {tree} {project} {branches} {peer}
```

Comment:
Lol at some point we should maybe bundle the source-related vars in some object :thinking:

Reply:
Sounds like a good idea. :sweat_smile: